src/lib/nvme_pt_ioctl.c

```diff
@@ -970,7 +979,6 @@ static int opal_get_msid(int fd, struct opal_device *dev, uint8_t *key, uint8_t
 		memcpy(key, dev->payload.tokens[4]->pos + jmp, *key_len);
 	} else {
 		SEDCLI_DEBUG_MSG("There is an error in parsing\n");
-		return ret;
```

Comment:
now in case of error opal_put_all_tokens will be called twice:
one in opal_generitc_get_column and second time in opal_get_msid

Reply:
yes. will fix it